### PR TITLE
Fix Novdoc conversion issues in Upstream Admin guide

### PR DESCRIPTION
**Pull request by sknorr**
via http://docserv.nue.suse.com/online-docs-reports/report-2019-03-02_18-01

@asettle / @dpopov could you specifically check the last commit that removes the empty sections for whether it does (additional) harm or is benign? I am not sure if these sections have always been empty and didn't want to investigate.

### CODE REVIEW

File: xml/admin-neutron-config-az.xml [hunk 1]
```diff
@@ -292,9 +292,6 @@ $ openstack network agent show 9632309a-2aa4-4304-8603-c4de02c4a55f
                         capacity left in the zone requested.</para>
       </note>
     </section>
-    <section>
-      <title>Availability zone aware scheduler</title>
-    </section>
```
Comment (asettle) [inline xml/admin-neutron-config-az.xml]: So this shouldn't be removed, it's just not formatting quite right. This needs to be the section title, and the Network Scheduler needs to be the sub title within that. See: https://docs.openstack.org/newton/networking-guide/config-az.html

File: xml/admin-ironic.xml [hunk 2]
```diff
@@ -3575,9 +3575,6 @@ URL</literal>.</para>
                   <screen>ironic port-create -n $NODE -a &lt;MAC_address_of_Ucs_server's_NIC&gt;</screen>
                 </section>
               </section>
-              <section>
-                <title>References</title>
-              </section>
```
Comment (asettle) [inline xml/admin-ironic.xml]: This is missing a link: https://docs.openstack.org/ironic/pike/admin/drivers/ucs.html#references

File: xml/admin-keystone.xml [hunk 3]
```diff
@@ -83729,9 +83729,6 @@ class Something(object):
             </varlistentry>
           </variablelist>
         </section>
-        <section xml:id="module-keystone_module-contents">
-          <title>Module contents</title>
```
Comment (asettle) [inline xml/admin-keystone.xml]: This one is particular weird. These sections should belong to the API content. I'll investigate further, this one is weird.

File: xml/admin-neutron-config-az.xml (hunk 1 above)
Comment (asettle) [reply xml/admin-neutron-config-az.xml]: Ugh hang on, that link is too many versions old.

Comment (asettle) [reply xml/admin-neutron-config-az.xml]: https://docs.openstack.org/neutron/pike/admin/config-az.html#availability-zone-aware-scheduler

File: xml/admin-keystone.xml (hunk 3 above)
Comment (dmpop) [reply xml/admin-keystone.xml]: So is it weird or is it weird weird?

Comment (asettle) [reply xml/admin-keystone.xml]: Weird weird. It's all API config content. How did that get converted with the admin guide? Removed here:  https://github.com/SUSE-Cloud/doc-cloud/pull/800/commits/4f79d3f128c377fe3429c2ea12c961b656ae1124

Comment (asettle) [reply xml/admin-keystone.xml]: Basically, everything from here: https://docs.openstack.org/keystone/pike/configuration/index.html
Got converted too.

File: xml/admin-ironic.xml (hunk 2 above)
Comment (asettle) [reply xml/admin-ironic.xml]: Although relaly this one doesn't matter

File: xml/admin-neutron-config-az.xml (hunk 1 above)
Comment (asettle) [reply xml/admin-neutron-config-az.xml]: This one should be better nested

**Comment by asettle** [comment]
@sknorr I think that last PR deals with it :) the other one can be deleted.

**Comment by sknorr** [comment]
Thanks a lot @asettle !

**Pull request merged**